tools/cldr-code/src/test/java/org/unicode/cldr/util/TestCLDRPaths.java

```diff
@@ -0,0 +1,50 @@
+package org.unicode.cldr.util;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+import java.io.File;
+import org.junit.jupiter.api.Test;
+
+public class TestCLDRPaths {
+
+    @Test
+    void TestCanUseArchiveDirectory() {
+        if (!canUseArchiveDirectory()) {
+            // We print this warning as part of the unit tests.
+            System.err.println(
+                    "WARNING: skipping using cldr-archive. Ideally, -DNO_CLDR_ARCHIVE=false and see <https://cldr.unicode.org/development/creating-the-archive>");
```

Comment:
Unclear message. Does it mean the following? 

Ideally you should set -DNO_CLDR_ARCHIVE=false **AND** create the archive as per <https://cldr.unicode.org/development/creating-the-archive>

Reply:
They are already opting out. The message is to suggest no longer opting out.

Reply:
double negatives are often confusing: HAS_CLDR_ARCHIVE=true would be better

Reply:
Except the default is false. 

How about: SKIP_CLDR_ARCHIVE=false as the default. You set it to true if you don't want to setup CLDR-archive.

Reply:
changes to HAS_CLDR_ARCHIVE=false per discussion